index.d.ts

```diff
@@ -1,2 +1,5 @@
+/// <reference types="typescript" />
 declare module '*.svg';
 declare module '*.css';
+declare module '*.jpg';
+declare module 'styled-components';
```

Comment:
Why did you declare styled-components as module?

Reply:
Otherwise Typescript can't identify the styled-components' type.

Reply:
That's not the way to go. In that case you should install the types of styled-components: https://www.npmjs.com/package/@types/styled-components.